### ISSUE TITLE
does not support decimal values? 

### ISSUE THREAD

**Issue by vance** [darul75/angular-awesome-slider#135]
Can't get it to do range 0-1 with step of .1

**Comment by darul75** [comment]
Put options attribute round:2 tell me it is fine, thx

**Comment by FlavorScape** [comment]
ah yes, that works. I would expect this variable to be called ```precision``` or something. I associate rounding (on a slider) with 'snapping' to positions (reducing granularity). 

Maybe add to readme

For values 0-1 use the round option to specify the precision to which to calculate the value.

thanks!

great library BTW I just wish there wasn't a bourbon dependency for the SCSS, but cheers for providing so many CSS options.

**Comment by darul75** [comment]
Yes your are true about precision attribute, I should rename it soon :+1: 

Someone has made a great work by porting all my less files to scss but they are not used today, only less files are used while generating css. What do you mean with the scss dependency, I do not know, does it come from...hmm no !!! I see know it is into bower config, I will take a look as it is not really used today I think.
